modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -2670,7 +2670,7 @@ def update_json_and_submit(updated_json_lambda)
                   service_period['separationLocationCode'] = separation_location_code
                   data = json.to_json
                   post submit_path, params: data, headers: auth_header
-                  expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
Here and below: should we make the test description match our assertion?

Reply:
shoot, yes good catch, fixing now

Reply:
@tycol7 [faa9b86](https://github.com/department-of-veterans-affairs/vets-api/pull/18220/commits/faa9b86f0fe777c7a56f3b699a5195eed7e4b985)